eth/backend.go

```diff
@@ -65,7 +65,7 @@ const (
 var (
 	jsonlogger = logger.NewJsonLogger()
 
-	datadirInUseErrNos = []uint{11, 32, 35}
+	datadirInUseErrnos = map[uint]struct{}{11: struct{}{}, 32: struct{}{}, 35: struct{}{}}
```

Comment:
This fix is nice and simple, so :+1:. However, if you have the time, please make this a `map[uint]bool` instead.
Empty structs are nice but hurt readbility in this case.